src/sql/src/normalize.rs

```diff
@@ -108,6 +108,20 @@ pub fn create_statement(
     }
 
     impl<'a, 'ast> VisitMut<'ast> for QueryNormalizer<'a> {
+        fn visit_expr_mut(&mut self, e: &'ast mut Expr) {
+            match e {
+                // Needs to be resolved to qualified name because of `ALTER
+                // <foo> RENAME...`.
+                Expr::QualifiedWildcard(i) => {
+                    let o = ObjectName(i.clone());
+                    match self.scx.resolve_item(o) {
+                        Ok(full_name) => *i = unresolve(full_name).0,
+                        Err(e) => self.err = Some(e),
+                    };
```

Comment:
This looks suspicious to me. What if `i` is an alias that is introduced in the query?